### PR TITLE
Fixes for the failing Unit tests - Create site.

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**
- Fixes for the failing unit tests in create Site.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

- Change the line with test in the `package.json` file to
`"test": "mocha ./utils/test/ut_create-site.js --exit"`
- Navigate to src/device-registry and `run npm test`

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/device-registry/utils/create-site.js [hunk 1]
```diff
@@ -222,6 +224,13 @@ const createSite = {
           }
           if (isEmpty(responseJSON.data)) {
             logElement("unable to list stations");
+            return {
```
Comment (Baalmart) [inline src/device-registry/utils/create-site.js:227]: Hi @BenjaminSsempala , can you shed some light on this specific change under` listWeatherStations`?

1. Can we ensure that it does not break anything else?
2. What other functions utilise it? Do they anticipate this addition in the way they respond to the output from this function?

Thanks!

File: src/device-registry/utils/create-site.js [hunk 2]
```diff
@@ -873,6 +882,8 @@ const createSite = {
   },
   formatSiteName: (name) => {
     try {
+      let nameWithoutWhiteSpace = name.replace(/\s/g, "");
```
Comment (Baalmart) [inline src/device-registry/utils/create-site.js:885]: thanks for this @BenjaminSsempala

File: src/device-registry/utils/distance.js [hunk 3]
```diff
@@ -174,41 +174,52 @@ const distance = {
...
-      ),
-      approximate_longitude: distance.radiansToDegrees(
-        approximateLongitudeInRadians
-      ),
-      approximate_distance_in_km,
-      bearing_in_radians: parseFloat(bearingInRadians),
-      provided_latitude: parseFloat(latitude),
-      provided_longitude: parseFloat(longitude),
-    };
+        );
```
Comment (Baalmart) [inline src/device-registry/utils/distance.js:199]: thanks @BenjaminSsempala for this!

File: src/device-registry/utils/create-site.js (hunk 1 above)
Comment (BenjaminSsempala) [reply src/device-registry/utils/create-site.js:227]: Thanks @Baalmart 
I've confirmed this

**Review by Baalmart — APPROVED** [review]

**Pull request merged**